gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenResource.java

```diff
@@ -626,14 +628,14 @@ public Response revoke(String token) {
         final String tokenId = getTokenId(token);
         if (isKnoxSsoCookie(tokenId)) {
           errorStatus = Response.Status.FORBIDDEN;
-          error = "SSO cookie (" + Tokens.getTokenIDDisplayText(tokenId) + ") cannot not be revoked." ;
+          error = "SSO cookie (" + Tokens.getTokenIDDisplayText(tokenId) + ") cannot not be revoked.";
           errorCode = ErrorCode.UNAUTHORIZED;
         } else if (triesToRevokeOwnToken(tokenId, revoker) || allowedRenewers.contains(revoker)) {
           tokenStateService.revokeToken(tokenId);
           log.revokedToken(getTopologyName(),
-              Tokens.getTokenDisplayText(token),
-              Tokens.getTokenIDDisplayText(tokenId),
-              revoker);
+                  Tokens.getTokenDisplayText(token),
```

Comment:
formatting change

Reply:
Not sure why it was the previous indentation. I guess someone is lining up the dots? That is not a convention that is standard.  That said, I don't know why this was changed at all. Likely some IDE shenanigans related to my migration to a new dev machine. These are all in the same method and I am working backwards through them. I ilke how the code reads with this formatting. If you can look at the code in context and think I should revert the changes then I will. We should also add to the checkstyle rules if that is the case though.